driver/debug/debug.go

```diff
@@ -42,6 +42,7 @@ func (d *Driver) Config() map[string]string {
 }
 
 // SetConfig sets configuration for this driver
-func (d *Driver) SetConfig(settings map[string]string) {
+func (d *Driver) SetConfig(settings map[string]string) error {
```

Comment:
Nice catch!
Do we have some tests to make sure the various drivers properly implement the Driver interface?
(of course, not blocking this PR).